apps/desktop-wallet/src/api/apiDataHooks/utils/useFetchFtList.ts

```diff
@@ -19,22 +16,8 @@ type FTListProps = {
 
 const useFetchFtList = (props?: FTListProps): FTList => {
   const networkId = useAppSelector(selectCurrentlyOnlineNetworkId)
-  const network = networkId === 0 ? 'mainnet' : networkId === 1 ? 'testnet' : networkId === 4 ? 'devnet' : undefined
 
-  const { data, isLoading } = useQuery({
-    queryKey: ['tokenList', { networkId }],
-    // The token list is essential for the whole app so we don't want to ever delete it. Even if we set a lower gcTime,
-    // it will never become inactive (since it's always used by a mount component).
-    ...getQueryConfig({ staleTime: ONE_DAY_MS, gcTime: Infinity, networkId }),
-    queryFn:
-      !network || props?.skip
-        ? skipToken
-        : () =>
-            network === 'devnet'
-              ? [ALPH]
-              : axios.get(getTokensURL(network)).then(({ data }) => (data as TokenList)?.tokens),
-    placeholderData: network === 'mainnet' ? mainnet.tokens : network === 'testnet' ? testnet.tokens : []
-  })
```

Comment:
Simply moved the code into a new `ftListQuery` function, because I wanted to reuse it in multiple places.